src/main/java/org/springframework/samples/petclinic/repository/jdbc/JdbcVisitRepositoryImpl.java

```diff
@@ -138,6 +140,21 @@ private MapSqlParameterSource createVisitParameterSource(Visit visit) {
         return visits;
     }
 
+    @Override
+    public List<Visit> findAllFutureVisits(LocalDate current_date) {
+        Date date = Date.valueOf(current_date);
+
+        List<JdbcPet> results = this.jdbcTemplate.query(
+            "SELECT id, name, birth_date, type_id, owner_id FROM pets",
+            new JdbcPetRowMapper());
+
+        List<Visit> futureVisits = this.jdbcTemplate.query(
+            "SELECT id as visit_id, visit_date, description, pet_id FROM visits" +
+                " WHERE visit_date > '"+ date +"'",
```

Comment:
use paramterized queries rather than string concatenation. For an example look at deleteById below

Reply:
Parameters added